BlueprintUI/Sources/Layout/Aligned.swift

```diff
@@ -2,8 +2,10 @@ import UIKit
 
 /// Aligns a content element within itself. The vertical and horizontal alignment may be set independently.
 ///
-/// The size of the content element is determined by calling `measure(in:)` on
-/// the content element – even if that size is larger than the wrapping element.
+/// When using alignment mode `.fill`, the content is scaled to the width or height of the `Aligned` element.
+///
+/// For other modes, the size of the content element is determined by calling `measure(in:)`
+/// on the content element – even if that size is larger than the wrapping element.
```

Comment:
@watt not going to change it here, but this bit "even if that size is larger than the wrapping element." seems a bit weird to me – is there a use case for this, or should we change the implementation to cap the size?

Reply:
I copied this from `Centered`. I think it's reasonable behavior, as it limits the responsibility of this element and favors composition (also a reason that I have mixed feelings about adding `fill` here instead of introducing a `Fill` element). If you want to cap the size of an element, composition with `ConstrainedSize` seems like a more scalable solution.